rmgpy/pdep/sls.py

```diff
@@ -46,6 +45,17 @@
 from rmgpy.pdep.me import generate_full_me_matrix, states_to_configurations
 from rmgpy.statmech.translation import IdealGasTranslation
 
+NO_JULIA = False
+try:
+    from diffeqpy import de
+    from julia import Main
+except Exception as e:
+    warnings.warn(
+        f"Unable to import Julia dependencies, original error: {str(e)}"
+        ". Master equation method 'ode' will not be available on this execution."
+    )
```

Comment:
Logging.info is used in the rmg.rms_reactor, so I think we should be consistent about the way we show this information

```suggestion
    logging.info(
        f"Unable to import Julia dependencies, original error: {str(e)}"
        ". Master equation method 'ode' will not be available on this execution."
    )
```